src/Exceptions/SchemaModelException.php

```diff
@@ -4,8 +4,13 @@
 
 class SchemaModelException extends SnapshotsException
 {
-    public static function create(string $table): self
+    public static function wrongCreateMethod(string $table): self
     {
-        return new self('You must use the `createForModel` method on the Snapshot schema builder to use the CopyModels auto_copier. You are currently using the `create` method for table `'.$table.'`.');
+        return new self('You must use the `createForModel` method on the Snapshot schema builder in your migrations to use History Tracking. You are currently using the `create` method for table `'.$table.'`.');
+    }
+
+    public static function notVersioned(string $model): self
+    {
+        return new self('Model `'.$model.'` must implement the `'.\Plank\Snapshots\Contracts\Versioned::class.'` interface to use History Tracking.');
     }
```

Comment:
I think this is not even true 😂... Pretty sure I made it so that any model can use history. Unversioned models will just always have all of their operations in the "working version".

Reply:
Ok. Fixed extracted the Exception and created tests for non-versioned history tracking.